components/movementsensor/rtkutils/config.go

```diff
@@ -0,0 +1,42 @@
+// Package rtkutils contains GPS-related code shared between multiple components. This file
+// describes data structures used to configure reading from NMEA devices.
+package rtkutils
+
+import (
+	"go.viam.com/rdk/resource"
+)
+
+// SerialConfig is used for converting Serial NMEA MovementSensor config attributes.
+type SerialConfig struct {
+	SerialPath     string `json:"serial_path"`
+	SerialBaudRate int    `json:"serial_baud_rate,omitempty"`
+
+	// TestChan is a fake "serial" path for test use only
+	TestChan chan []uint8 `json:"-"`
+}
+
+// I2CConfig is used for converting Serial NMEA MovementSensor config attributes.
+type I2CConfig struct {
+	I2CBus      string `json:"i2c_bus"`
+	I2CAddr     int    `json:"i2c_addr"`
+	I2CBaudRate int    `json:"i2c_baud_rate,omitempty"`
+}
```

Comment:
perhaps these to RTKGPSSerialand RTKI2CCofnig since there are other components with the same config names that are exported.